gradle/libs.versions.toml

```
@@ -1,15 +1,15 @@
 [versions]
 accompanist = "0.33.0-alpha"
-activityCompose = "1.8.0"
-androidGradlePlugin = "8.1.2"
+activityCompose = "1.8.2"
+androidGradlePlugin = "8.2.0"
 coil = "2.5.0"
-dataProvider = "0.7.7"
-kotlinPlugin = "1.9.10"
+dataProvider = "0.8.0"
+kotlinPlugin = "1.9.21"
 detekt = "1.23.3" # https://github.com/detekt/detekt
 kotlinxSerializationJson = "1.6.0"
 ktorVersion = "2.3.3"
```

```suggestion
ktorVersion = "2.3.7"
```

https://github.com/ktorio/ktor/releases